### PR TITLE
os: update vmware guide

### PR DESCRIPTION
**Pull request by crawford**
Users should be using the standard VMware images. This suggests that
they use the OVF image so it can be directly imported into ESXi.

### CODE REVIEW

**Comment by crawford** [comment]
I haven't yet tested this guide.

File: os/booting-on-vmware.md [hunk 1]
```diff
@@ -1,71 +1,68 @@
...
+The channel is selected based on the URLs below. Simply replace `alpha` with
+`beta` or `stable`. Read the [release notes][release notes] for specific
+features and bug fixes in each channel.
 
 ```sh
-curl -LO http://alpha.release.core-os.net/amd64-usr/current/coreos_production_vmware_insecure.zip
-unzip coreos_production_vmware_insecure.zip -d coreos_production_vmware_insecure
-cd coreos_production_vmware_insecure
-open coreos_production_vmware_insecure.vmx
+curl -LO http://alpha.release.core-os.net/amd64-usr/current/coreos_production_vmware_ova.ovf
```
Comment (robszumski) [inline os/booting-on-vmware.md]: Do you want this to change based on the tab selected? If so, we need to add the tabs. If not, we should probably default to stable.

**Comment by robszumski** [comment]
Text looks good to me other than the tab question.

File: os/booting-on-vmware.md (hunk 1 above)
Comment (crawford) [inline os/booting-on-vmware.md]: I'll default to stable.

File: os/booting-on-vmware.md [hunk 2]
```diff
@@ -1,71 +1,68 @@
...
+The channel is selected based on the URLs below. Simply replace `stable` with
+`alpha` or `beta`. Read the [release notes][release notes] for specific
+features and bug fixes in each channel.
 
 ```sh
-curl -LO http://alpha.release.core-os.net/amd64-usr/current/coreos_production_vmware_insecure.zip
-unzip coreos_production_vmware_insecure.zip -d coreos_production_vmware_insecure
-cd coreos_production_vmware_insecure
-open coreos_production_vmware_insecure.vmx
+curl -LO http://stable.release.core-os.net/amd64-usr/current/coreos_production_vmware_ova.ovf
```
Comment (marineam) [inline os/booting-on-vmware.md]: .ova not .ovf

Comment (crawford) [inline os/booting-on-vmware.md]: oops, both are needed

Comment (marineam) [inline os/booting-on-vmware.md]: no, .ova includes the ovf and vmdk. If you fetch the ovf you do need to fetch the vmdk.

**Pull request merged**